scripts/linux/build-appimage.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash -ex
+#!/usr/bin/env bash
+
+set -ex
 
 # use RAM disk if possible
 if [ "$CI" == "" ] && [ -d /dev/shm ]; then
```

Comment:
the use of `&&` here is unlike in other places, where it's inside the `[`, `]`, any comment on this?

Reply:
If they behave the same they should be made consistent. (inside the braces)

Reply:
The `$CI` line could also use the `-z` treatment done in the second to last commit, right?